src/components/Text/Text.stories.tsx

```diff
@@ -74,6 +75,10 @@ export const WithControls: Story = {
         Nintendo 3DS, Animal Crossing: Amiibo Festival for Wii U, and Animal
         Crossing: Pocket Camp for mobile devices.
       </Text>
+      <Text {...args}>
+        As of June 2024, the Animal Crossing franchise has sold over 81,450,000
```

Comment:
You can use the function for the `81,450,000` as well. But you should set both of those examples as const variables first and then use the values within the `Text` component.
```
const totalSold = formatNumber(81450000);
const retailPrice = formatNumber(10, 1500);
```